packages/design-system/src/components/Autocomplete/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -26,6 +26,7 @@ exports[`Autocomplete renders Autocomplete component 1`] = `
       autocomplete="off"
       class="ds-c-field"
       id="autocomplete_1"
+      labelid="autocomplete_label_2"
```

Comment:
Is this possibly a symptom of the `TextInput` including an attribute that it shouldn't?